tests/ipv6_test.go

```diff
@@ -291,15 +339,42 @@ func TestIPv6_AppendRow(t *testing.T) {
 	if !col.Row(3, false).(net.IP).Equal(ip) {
 		require.Failf(t, "Invalid result of AppendRow", "Added %q instead of %q", col.Row(3, false), ip)
 	}
-}
 
-func TestIPv6_Append(t *testing.T) {
-	ips := getTestIPv6()
+	// appending [16]byte
+	require.NoError(t, col.AppendRow(netip.MustParseAddr(ip.String()).As16()))
+	require.Equal(t, 5, col.Rows(), "AppendRow didn't add IP")
+	if !col.Row(4, false).(net.IP).Equal(ip) {
+		require.Failf(t, "Invalid result of AppendRow", "Added %q instead of %q", col.Row(4, false), ip)
+	}
+
+	// appending proto.IPv6
+	require.NoError(t, col.AppendRow(proto.IPv6(netip.MustParseAddr(ip.String()).As16())))
+	require.Equal(t, 6, col.Rows(), "AppendRow didn't add IP")
+	if !col.Row(5, false).(net.IP).Equal(ip) {
+		require.Failf(t, "Invalid result of AppendRow", "Added %q instead of %q", col.Row(5, false), ip)
+	}
```

Comment:
```suggestion
        assert.Truef(t, !col.Row(5, false).(net.IP).Equal(ip), "Added %q instead of %q", col.Row(5, false), ip)
```

Reply:
or there might be a helper func that does following

```go
func assertRowColumnEqualToIP(t *testing.T, value any, ip net.IP) {
	require.IsType(t, net.IP{}, value, "Invalid type of column value. net.IP expected.")

	assert.Truef(t, value.(net.IP).Equal(ip), "%q is not equal to IP %q", value, ip)
}
```

```suggestion
	assertRowColumnEqualToIP(t, col.Row(5, false), IP)
```

it can be reused in duplicated code